tests/chart_tests/helm_template_generator.py

```diff
@@ -58,6 +59,14 @@ def create_validator(api_version, kind, kube_version="1.21.0"):
 
 def validate_k8s_object(instance, kube_version="1.21.0"):
     """Validate the k8s object."""
+    # Skip PostgreSQL chart
```

Comment:
Why do we skip the postgres chart?

Reply:
@danielhoherd 
```
namespace = '123'

    @pytest.mark.parametrize("namespace", ["abc", "123", "123abc", "123-abc"])
    def test_namespace_names(namespace):
        """Test various namespace names to make sure they render correctly in templates"""
>       render_chart(namespace=namespace)

tests/chart_tests/test_default_chart.py:14: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/chart_tests/helm_template_generator.py:129: in render_chart
    validate_k8s_object(k8s_object, kube_version=kube_version)
tests/chart_tests/helm_template_generator.py:64: in validate_k8s_object
    validate.validate(instance)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = Draft7Validator(schema={'$schema': 'http://json/-...a.org/schema#', 'description': 'Secret holds...etSize bytes.', 'pro...t...d#types-kinds', 'enum': ['Secret'], 'type': ['string', 'null']}, ...}, 'type': 'object', ...}, format_checker=None)
args = ({'apiVersion': 'v1', 'data': {'postgresql-password': 'cG9zdGdyZXM='}, 'kind': 'Secret', 'metadata': {'labels': {'app....ame': 'postgresql', 'helm.sh/chart': 'postgresql-10.5.3'}, 'name': 'RELEASE-NAME-postgresql', 'namespace': 123}, ...},)
kwargs = {}, error = <ValidationError: "123 is not of type 'string', 'null'">

    def validate(self, *args, **kwargs):
        for error in self.iter_errors(*args, **kwargs):
>           raise error
E           jsonschema.exceptions.ValidationError: 123 is not of type 'string', 'null'
E           
E           Failed validating 'type' in schema['properties']['metadata']['properties']['namespace']:
E               {'description': 'Namespace defines the space within which each name '
E                               'must be unique. An empty namespace is equivalent to '
E                               'the "default" namespace, but "default" is the '
E                               'canonical representation. Not all objects are '
E                               'required to be scoped to a namespace - the value of '
E                               'this field for those objects will be empty.\n'
E                               '\n'
E                               'Must be a DNS_LABEL. Cannot be updated. More info: '
E                               'http://kubernetes.io/docs/user-guide/namespaces',
E                'type': ['string', 'null']}
E           
E           On instance['metadata']['namespace']:
E               123

venv/lib/python3.9/site-packages/jsonschema/validators.py:247: ValidationErro
```
because of ^

inspired from OSS chart by @jedcunningham 

https://github.com/apache/airflow/blob/07fe9e8c6acf8826ac1dcaf2ad8c94297a0f0c24/chart/tests/helm_template_generator.py#L95-L96

Reply:
So this is to work around a casting problem where `str('123')` is turned into `int(123)`. Is there a way to skip only strings that can be cast as ints ?

Reply:
@jedcunningham any ideas? @danielhoherd i think it's okey to be on the same page with airflow OOS chart, so we can sync up code bases easily.

Reply:
I'm not sure I follow what you're suggesting @danielhoherd.

Ultimately opening a PR to bitnami to fix quoting on namespace names is probably the right way to solve this long term.

Reply:
I think it's good to keep in step with OSS, but I think we need to document why we are doing things that don't make logical sense without additional context. Having a comment would be good. Also it would be good to open an issue upstream.

Reply:
@danielhoherd fixed. PTAL